wolfcrypt/src/sp_sm2_arm32.c

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
please add license header to these dummy files

Reply:
Added